index.html

```diff
@@ -8,23 +8,34 @@
     <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;600;700&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@600&display=swap" rel="stylesheet">
     <title>Joseph Kwesi Kwentsir Portfolio</title>
 </head>
 <body>
+    <div class="background-image">
     <header class="page-header">
+        <!--Mobile Version navigation-->
       <nav class="nav-header"> 
-          <ul>
+          <ul class="mobile">
               <li><a href="#" class="welcome"> Welcome </a></li>
-              <li> <a href="#"> <img src="./images/Union.svg" alt=""/></a></li>
+              <li> <a href="#" class="union"> <img src="./images/Union.svg" alt=""/></a></li>
+          </ul>
+          <ul class="desktop">
+              <li><a href="#" class="desktop-menu"> Portfolio </a></li>
+              <li><a href="#" class="desktop-menu"> About </a></li>
+              <li><a href="#" class="desktop-menu"> Contact </a></li>
```

Comment:
- [ ] As already pointed out by the previous reviewer, please make sure the headline text is centered to match the original design

### Your Design
![image](https://user-images.githubusercontent.com/43995830/160255779-4410abcd-2574-4c71-8fbf-bd28594e08ec.png)

### Original Design
![image](https://user-images.githubusercontent.com/70269507/160253679-c56266e2-855b-4054-88dd-2f7d22525ac2.png)